### ISSUE TITLE
Is it possible to support png format？

### ISSUE THREAD

**Issue by terrellgf** [bitbank2/epd_image#1]
\# ./epd_image/epd_image --BWR --DITHER low-battery.png out.h
Unrecognized file format. For now, only BMP and JPEG are supported

**Comment by bitbank2** [comment]
Yes, I can add PNG support. I haven't written a single C file solution, but I can use my PNGdec library.
